miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/StudyRestController.java

```diff
@@ -0,0 +1,78 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller.rest;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Study;
+import uk.ac.bbsrc.tgac.miso.core.util.PaginatedDataSource;
+import uk.ac.bbsrc.tgac.miso.core.util.PaginationFilter;
+import uk.ac.bbsrc.tgac.miso.dto.DataTablesResponseDto;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.dto.StudyDto;
+import uk.ac.bbsrc.tgac.miso.service.StudyService;
+
+@Controller
+@RequestMapping("/rest/study")
+public class StudyRestController extends RestController {
+  private final JQueryDataTableBackend<Study, StudyDto> jQueryBackend = new JQueryDataTableBackend<Study, StudyDto>() {
+    @Override
+    protected StudyDto asDto(Study model) {
+      return Dtos.asDto(model);
+    }
+
+    @Override
+    protected PaginatedDataSource<Study> getSource() throws IOException {
+      return studyService;
+    }
+  };
+
+  @Autowired
+  private StudyService studyService;
+
+  @RequestMapping(value = "/dt", method = RequestMethod.GET, produces = "application/json")
+  @ResponseBody
+  public DataTablesResponseDto<StudyDto> dataTable(HttpServletRequest request, HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
+    return jQueryBackend.get(request, response, uriBuilder);
+  }
+
+  @RequestMapping(value = "/dt/project/{id}", method = RequestMethod.GET, produces = "application/json")
+  @ResponseBody
+  public DataTablesResponseDto<StudyDto> dataTableByProject(@PathVariable("id") Long id, HttpServletRequest request,
+      HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
+    return jQueryBackend.get(request, response, uriBuilder, PaginationFilter.project(id));
+  }
+
+  @RequestMapping(value = "/{id}", method = RequestMethod.DELETE, produces = "application/json")
+  @ResponseBody
+  public ResponseEntity<?> delete(@PathVariable("id") Long id, HttpServletRequest request,
+      HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
+    Study study = studyService.get(id);
+    if (study == null) {
+      return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+    studyService.delete(study);
+    return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
Could change to `NO_CONTENT` to be more accurate (though we use `OK` most places anyway). Could use `@ResponseStatus` instead of returning `ResponseEntity<?>`